ParkJInjoo/app.js

```diff
@@ -0,0 +1,41 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("combined"));
+
+dotenv.config();
+
+const { DataSource } = require("typeorm");
```

Comment:
- 모듈 import 순서
    - import 순서 컨벤션에 맞게 수정해주세요! 사소한 부분이라 놓치기 쉽지만 처음 코드를 작성할 때부터 신경쓰고 작성하게 되면 추후에 관리하기도 용이하며 가독성도 좋아집니다.
    - 다음과 같은 순서로 import 해줍니다.
        - Bulit-in 패키지
        - 3rd-party 패키지
        - custom 패키지
        
        ```jsx
        //// Built-in package
        const http = require('http')
        
        //// 3rd-party package
        require('dotenv').config();
        const express = require('express');
        const cors    = require('cors');
        const morgan  = require('morgan');
        const { DataSource } = require('typeorm');
        
        //custom package
        const api = require('./../api')
        
        const app = express();
        ```
```suggestion
const http = require("http");

require("dotenv").config()
const express = require("express");
const cors = require("cors");
const morgan = require("morgan");
const { DataSource } = require("typeorm");

app = express();

app.use(express.json());
app.use(cors());
app.use(morgan("combined"));
```

Reply:
안녕하세요 상웅님! 진주님께 해주신 리뷰에서  suggested change를 보다가 `const http = require("http");`를 위에 추가해주신 것을 확인하였는데, 이 http 모듈이 꼭 필요한 것인가요? express 프레임워크가 http 모듈을 대체하는 것 아닌가요?